src/main/kotlin/com/proudmuslim/andromeda/CreeperHeartItem.kt

```diff
@@ -8,12 +8,18 @@ import net.minecraft.item.Item
 import net.minecraft.util.Hand
 
 class CreeperHeartItem(settings: Settings): Item(settings) {
+
+    private var using: Boolean = true
```

Comment:
que?

Reply:
fires twice so we only make it fire every other time it's called

Reply:
ah